README.md

```diff
@@ -0,0 +1,57 @@
+# Bubble sort with ruby
+
+>  Welcome to my first ruby project. In this project i will take an array and bubble sort it. The patern will check the first number vs second then it will swap or move to next number and repair. 
+
+![screenshot](./screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Ruby
+- Rubocop
+
+## Getting Started
+
+Clone directory and run with Ruby. Open terminal inside project folder and type "ruby hello.rb" and hit enter.
+
+
+### Prerequisites
+
+Ruby installed on your computer to be able to read/run this file. 
+
+### Setup
+
+If your using Arch you can type in terminal.. 
+
+sudo pacman -S ruby --noconfirm 
+
+to install. if your OS is diffrent check out the link bellow.
+
+https://www.ruby-lang.org/en/downloads/
```

Comment:
This part is not descriptive of your project. You don't have to use the prerequisites but you should give the user some guidance to run your methods.